### PR TITLE
EXUI-424-added appinsights logs for task completion

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EXUI-424

### Change description ###
added appinsights logd for task completion

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by danlysiak — DISMISSED** [review]

**Review by arun-hmcts — DISMISSED** [review]

File: api/.env.defaults [hunk 1]
```diff
@@ -8,7 +8,7 @@ NODE_CONFIG_ENV=development
 ## SECRETS - Note: these values need to be faked so that tests pass in the pipeline
 #IDAM_SECRET=AAA
 #S2S_SECRET=AAA
-APPINSIGHTS_INSTRUMENTATIONKEY=AAA
+APPINSIGHTS_INSTRUMENTATIONKEY=5f607ccb-4ab5-4c9d-a4b9-801868bdf698
```
Comment (danlysiak) [inline api/.env.defaults:11]: This should be reverted before any merge

File: src/work-allocation/services/work-allocation-task.service.spec.ts [hunk 2]
```diff
@@ -36,25 +36,25 @@ describe('WorkAllocation', () => {
     it('completeTask should make correct api call', () => {
       const service = new WorkAllocationTaskService(mockHttpService);
       service.completeTask('123456', true);
-      expect(mockHttpService.post).toHaveBeenCalledWith('/workallocation/task/123456/complete', { hasNoAssigneeOnComplete: true });
+      expect(mockHttpService.post).toHaveBeenCalledWith('/workallocation/task/123456/complete', { hasNoAssigneeOnComplete: true, hasActionTriggeredManually: undefined });
```
Comment (danlysiak) [inline src/work-allocation/services/work-allocation-task.service.spec.ts:39]: Assuming the API is OK with being passed `undefined`?

**Review by danlysiak — APPROVED** [review]
Approved with comments

File: api/.env.defaults (hunk 1 above)
Comment (RiteshHMCTS) [reply api/.env.defaults:11]: yes sure @danlysiak

**Review by kasi-subramaniam — DISMISSED** [review]

**Review by kasi-subramaniam — DISMISSED** [review]

**Review by olusegz07 — DISMISSED** [review]

**Review by mwaheedanjum — APPROVED** [review]

**Review by olusegz07 — APPROVED** [review]

**Pull request merged**